Distance_3/test/Distance_3/test_distance_3.cpp

```diff
@@ -221,6 +221,17 @@ struct Test {
     check_squared_distance (L(p(2, -4,  3), p( 3,-8, 4)), Pl(0, 1, 0, 0), 0);
   }
 
+  void Pl_Pl()
+  {
+    std::cout << "Plane - Plane\n";
+    Pl p1(12, 1, 4, 0);
+    typename K::Vector_3 v = -p1.orthogonal_vector();
+    v /= CGAL::sqrt(v.squared_length());
+    Pl p2 = Pl(p1.point()+6*v, -p1.orthogonal_vector());
+    check_squared_distance (p1,p2, 36);
```

Comment:
I'm surprised this works, you better construct yourself a plane, rather rely on inexact sqrt.